int32 first_guess
int32 second_guess
int32 third_guess
